be/src/common/config.h

```diff
@@ -516,6 +520,7 @@ CONF_Int32(flush_thread_num_per_store, "2");
 // config for tablet meta checkpoint
 CONF_mInt32(tablet_meta_checkpoint_min_new_rowsets_num, "10");
 CONF_mInt32(tablet_meta_checkpoint_min_interval_secs, "600");
+CONF_mInt32(generate_tablet_meta_checkpoint_tasks_interval_secs, "10");
```

Comment:
Would produce checkpoint task every 10 seconds cause BE do checkpoint task too frequently? better to keep the behavior similar as before to set it to 600?